executor/local/stage.go

```diff
@@ -90,6 +91,7 @@ func (c *client) ExecStage(ctx context.Context, s *pipeline.Stage, m *sync.Map)
 		//
 		// https://pkg.go.dev/github.com/go-vela/worker/internal/step#Skip
 		if step.Skip(_step, c.build, c.repo) {
+			logrus.Infof("Skipping step %s due to ruleset", _step.Name)
```

Comment:
disclaimer i havent tested the PR to see what the logging actually looks like mid-execution. but we usually print to stdout during pipeline execution, should we do that here to follow the pattern?

Reply:
It looks the same as all the `init` step logs: 
```sh
vela exec pipeline 
DEBU[0000] validating pipeline configuration            
DEBU[0000] Creating registry clients from CLI configuration 
...
[step: init] > Preparing service images...
[step: init] > Preparing stage images...
[step: init] > Preparing step images...
DEBU[0000] streaming build logs                         
[step: init] $ docker image inspect golang:latest
sha256:2159148dcc081245165b2aa99fc5a94ca9818bece66839d8eb11c9335ae9e688
...
[step: docker-build] INFO[0000] Skipping push to container registry due to --no-push flag 
INFO[0004] Skipping step notify_slack due to ruleset 
[step: random] > Streaming container 'step_EastonCrupper_myvela_1_random'...
```
Think that's decent enough?

Reply:
ah sweet, the example is great. it views more like a log from the worker process controlling the execution flow, i like it!